src/app/main-page/presentation/toolbar-action/toolbar-action.component.ts

```diff
@@ -32,15 +35,23 @@ export class ToolbarActionComponent implements OnChanges {
     @Input() tracingActive: boolean;
     @Input()
     set graphSettings(value: GraphSettings) {
-        this.selectedMapTypeOption = "" + value.mapType;
+        const { mapType, tileServer } = value;
+
+        if (mapType === MapType.TILES_ONLY) {
+            this.selectedOption = "" + tileServer;
+        } else {
+            this.selectedOption = "" + mapType;
+        }
+
```

Comment:
Here you don't need the `"" +` anymore because these enum values are strings

Reply:
- [ ] remove string coersion